### ISSUE TITLE
[speedtest] Probable mismatch between thing type label/description and binding label/description

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-addons#15267]
Probable mismatch between thing type label/description and binding label/description.
In properties file
```
# add-on

addon.speedtest.name = Speedtest Binding
addon.speedtest.description = Binding for Ookla Speedtest (https://www.speedtest.net/)

# thing types

thing-type.speedtest.speedtest.label = Speedtest Binding
thing-type.speedtest.speedtest.description = Binding for Ookla Speedtest (https://www.speedtest.net/)
```
and in thing-types.xml
```
	<thing-type id="speedtest">
		<label>Speedtest Binding</label>
		<description>Binding for Ookla Speedtest (https://www.speedtest.net/)</description>
```

**Comment by lolodomo** [comment]
@MikeTheTux for analysis

**Comment by MikeTheTux** [comment]
@lolodomo, I'm not sure, if I got your point.

Are you asking for a change like:

```
# add-on

addon.speedtest.name = Speedtest Binding
addon.speedtest.description = Binding for Ookla Speedtest (https://www.speedtest.net/)

# thing types

thing-type.speedtest.speedtest.label = Speedtest Thing
thing-type.speedtest.speedtest.description = Instance of an Ookla Speedtest (https://www.speedtest.net/)
```

```
	<thing-type id="speedtest">
		<label>Speedtest Thing</label>
		<description>Instance of an Ookla Speedtest (https://www.speedtest.net/)</description>
```

**Comment by lsiepel** [comment]
i guess this can be closed as it seems to be fixed by #15278 ?

**Comment by lolodomo** [comment]
Correct
